demo/api/src/db/migrations/Migration20230302145445.ts

```diff
@@ -0,0 +1,13 @@
+import { Migration } from "@mikro-orm/migrations";
+
+export class Migration20230302145445 extends Migration {
```

Comment:
This migration should be in the library, which is perfectly fine, as we don't use the attached documents